### PR TITLE
Update External link buttons on No results page

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Related to #1187 by @obulat

## Description

This PR updates the external sources links buttons to a new variant and `medium` size.
The Figma file for these buttons is being updated, but we confirmed with @fcoveram that the buttons should use `bordered-gray` variant, `medium` size and have a 16px gap between them.

## Testing Instructions

Search for "non-existent-query" and see the "No results" page. Check that the buttons look correct and the CI passes.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/components/VErrorSection/VNoResults.vue [hunk 1]
```diff
@@ -13,8 +13,9 @@
         :key="source.name"
         as="VLink"
         :href="source.url"
-        variant="secondary-bordered"
-        class="label-bold !w-full justify-between text-dark-charcoal md:!w-auto md:justify-start md:gap-x-2"
+        variant="bordered-gray"
+        size="medium"
+        class="label-bold !w-full justify-between text-dark-charcoal md:!w-auto"
         show-external-icon
         :external-icon-size="4"
```
Comment (fcoveram) [inline frontend/src/components/VErrorSection/VNoResults.vue:20]: The icon should update the size to `24px` to match the button base component.

**Review by fcoveram — CHANGES_REQUESTED** [review]
I noticed the following
* Right padding is incorrect. It should be `8px` as the [button base](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=4584-11487&t=bbML8jXgIu7SjMzF-4).
* In `sm` button should not be full width.

**Review by fcoveram — APPROVED** [review]
It looks great ✨ 🚀

**Review by dhruvkb — APPROVED** [review]
This looks great. LGTM!

**Pull request merged**